backend/src/main/java/de/sambam/gamenighttracker/controller/UserController.java

```diff
@@ -24,11 +22,19 @@ public UserController(UserService userService) {
         this.userService = userService;
     }
 
-    @GetMapping({"/games"})
-    public List<Game> gameList() {
-        return userService.listAllGames("1")
-                .orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "User not found"));
+    @GetMapping("/games")
+    public List<Game> getGameList() {
+        return userService.listAllGames("1");
     }
 
+    @GetMapping("/gamesessions")
+    public List<GameSession> getGameSessionsList() {
+        return userService.listAllSessions("1");
+    }
+
+    @GetMapping("/players")
+    public List<PlayerDto> getPlayerList() {
```

Comment:
usually DTO are used to consume incoming data e.g. an API or request and not returned from your application 
I would expect a "regular" object as Controller response